docs/DOCS_GUIDELINES.MD

```diff
@@ -0,0 +1,40 @@
+# Documentation guidelines
+
+## Best practices
+
+- Check the spelling and grammar, even if you have to copy and paste from an external source.
+- Use simple sentences. Easy-to-read sentences mean the reader can quickly use the guidance you share.
+- Try to express your thoughts in a concise and clean way.
+- Leave a space between the acronyms ADR and ICS and the corresponding number (e.g. ADR 008 and ICS 27).
+- Don't abuse `code` format when writing in plain English.
```

Comment:
What does this mean? Also I find `abuse` to be strong language. Maybe `overuse` is better?